pkg/k8shandler/curation.go

```diff
@@ -49,7 +49,7 @@ func CreateOrUpdateCuration(cluster *logging.ClusterLogging) (err error) {
 		retryErr := retry.RetryOnConflict(retry.DefaultRetry, func() error {
 			if exists, cluster := utils.DoesClusterLoggingExist(cluster); exists {
 				if !reflect.DeepEqual(curatorStatus, cluster.Status.Curation.CuratorStatus) {
-					logrus.Infof("Updating status of Curator")
+					logrus.Info("Updating status of Curator")
```

Comment:
I think it is useful to print this, but not more than once if looping in the RetryOnConflict